js/example-frontend-js/build.gradle

```diff
@@ -5,6 +5,24 @@
 apply plugin: 'kotlin-dce-js'
 apply from: rootProject.file('gradle/node-js.gradle')
 
+// Workaround resolving new Gradle metadata with kotlin2js
+// TODO: Remove once plugin is fixed
```

Comment:
...and how do we know that the plugin is fixed?

Reply:
Created issue